src/mem/port.hh

```diff
@@ -80,18 +80,27 @@ class MasterPort : public Port, public AtomicRequestProtocol,
     SimObject &owner;
 
   public:
+    /**
+     * @ingroup api_port
+     * @{
+     */
     MasterPort(const std::string& name, SimObject* _owner,
```

Comment:
FYI, this is going to conflict with https://gem5-review.googlesource.com/c/public/gem5/+/32308. We want to make sure that is merged first. I don't want to define Master/Slave in the API officially.

Reply:
Sure, it's gonna be a huge merge conflict. I'd wait until the terminology change to be done before pushing this to gerrit.